qiskit/result/result.py

```diff
@@ -170,23 +170,47 @@ def get_counts(self, experiment=None):
                 according to the registers in circuit (e.g. ``0100 1110``).
                 The string is little-endian (cr[0] on the right hand side).
 
+            List[dict[str:int]]: a list of dictionaries for each experiment
```

Comment:
Can you combine this with the paragraph above? Something like `dict[str:int] or List[dict[str:int]]: a dictionary or a list of dictionaries ...`

Reply:
Sure I will do this together with the other review